content/doc/administration/requirements/java.adoc

```diff
@@ -5,29 +5,26 @@ title:  Java requirements
 
 There are separate run and job execution requirements for Jenkins installations.
 
-## Running Jenkins
+Jenkins requires Java 11 or 17 since Jenkins 2.357 and LTS 2.361.1.
+link:/blog/2022/06/28/require-java-11/[Read more about this in the announcement blog post.]
+Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"].
 
-Java 11 is required for running modern versions of Jenkins.
 Upgrading an existing Jenkins setup from Java 8 to Java 11?
 Refer to the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
```

Comment:
Should this be an admonition block (NOTE)?

Reply:
Added admonition block for this information.